.settings.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings>
```

Comment:
;) This was added as part of later commits in this PR to fix pointing to spring repos.